server/master/pom.xml

```diff
@@ -72,6 +72,11 @@
       <groupId>org.apache.zookeeper</groupId>
       <artifactId>zookeeper</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.gaul</groupId>
+      <artifactId>modernizer-maven-annotations</artifactId>
+      <optional>true</optional>
+    </dependency>
```

Comment:
This will require adding the maven-dependency-analyzer to the build.  I am not sure I want to open that can of worms in this PR.  I can add that as a follow on.

Reply:
Nevermind.  This can be done without adding the dependency analyzer.  I missed the second dependency, outside the dependencyManagement section in the master pom.

Reply:
Oh, I didn't realize this, but you can just add the dependency as optional to the parent pom, and not care about the dependency analysis... since we're not doing that in the build for 1.x anyway.